src/main/java/com/bfrisco/benchants/utils/EconomyCharge.java

```diff
@@ -0,0 +1,7 @@
+package com.bfrisco.benchants.utils;
+
+public class EconomyCharge {

```

Comment:
Sorry, this might be bad practice, i am wanting to POSSIBLY add a dependency to vault but i havent got around to actually doing that.

Reply:
Ahh OK. Since we will have a command that removes money from players balances, vault should be used. The readme of this repo explains how to use it quite nicely:

https://github.com/MilkBowl/VaultAPI

Reply:
Im still on the fence about this though, would rather just imbue the items myself or better yet use a custom item that imbues it instead, would be more immersive and wouldnt make us depend on vault for continued use